### PR TITLE
test: add test cases about filter

### PR DESCRIPTION
**Pull request by tzssangglass**
1.  add test cases about filter
2. fix bug about filter chain

### CODE REVIEW

**Comment by tzssangglass** [comment]
@dmsolr @spacewander pls review

**Review by dmsolr — APPROVED** [review]

File: runner-core/src/main/java/org/apache/apisix/plugin/runner/handler/A6ConfigHandler.java [hunk 1]
```diff
@@ -65,6 +64,10 @@ private PluginFilterChain createFilterChain(Req req) {
                 logger.error("receive undefined filter: {}, skip it", conf.name());
                 continue;
             }
+            if (chainFilters.contains(filter)) {
+                logger.error("skip the same filter: {}", conf.name());
```
Comment (nic-chen) [inline runner-core/src/main/java/org/apache/apisix/plugin/runner/handler/A6ConfigHandler.java:68]: could we use a lower level to log ?

Comment (tzssangglass) [reply runner-core/src/main/java/org/apache/apisix/plugin/runner/handler/A6ConfigHandler.java]: update

**Review by nic-chen — APPROVED** [review]

**Comment by nic-chen** [comment]
@dmsolr 
Please help review, I am not familiar with the specific logic.
If there is OK, we could merge it.

**Review by dmsolr — APPROVED** [review]
LGTM

**Pull request merged**